packages/containers/psammead-timestamp-container/src/index.stories.jsx

```diff
@@ -122,13 +123,17 @@ storiesOf('Containers|TimestampContainer', module)
       // eslint-disable-next-line react/prop-types
       componentFunction: ({ locale, script, service }) => {
         updateAllTimestamps();
-
         const storyTimestamp = select(
           'Timestamp',
           timestamps,
           timestamps['Fixed (27 Aug 2019)'],
         );
-
+        const getAltCalendar = () => {
```

Comment:
If this is going to be extended, I'd usually recommend writing these mapper-type functions as hashmaps where possible:

```
const getAltCalendar = () => ({
  fa: jalaali,
  ps: jalaali
}[locale]);
```

Reply:
That's a good idea as more calendars could be added in the future